jsettlers.common/src/test/java/jsettlers/common/map/shapes/MapRectangleTest.java

```diff
@@ -0,0 +1,50 @@
+package jsettlers.common.map.shapes;
+
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+
+public class MapRectangleTest {
```

Comment:
Map rectangle is a rectangle on screen - so it is not a rectangle in map coordinates.

Reply:
That means the assumptions about the assertions are wrong? Should I remove the test?

Reply:
It might be the case that the test is wrong. It would be best if you can fix it.

Reply:
It might be the case that the test is wrong. It would be best if you can fix it.

Reply:
The tests are passing (also on master except for containsMax() because of an inconsistent usage of < or <= when checking the containment). So I don't understand what should be fixed.